x11-libs/libdlo/libdlo-0.1.2.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="A fully open source driver which supports all DisplayLink devices"
+HOMEPAGE="https://libdlo.freedesktop.org/wiki/"
+SRC_URI="https://people.freedesktop.org/~berniet/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="static-libs test-program"
+
+RDEPEND="virtual/libusb:0="
+
+DEPEND="${RDEPEND}"
+
+DOCS=( "AUTHORS" "ChangeLog" "Guide-v104.pdf" "README" )
+
+src_prepare() {
+	default
+
+	# AM_PROG_AR must be defined or automake will fail with:
+	# archiver requires 'AM_PROG_AR' in 'configure.ac'.
+	sed -i -e '/AC_PROG_CC/a AM_PROG_AR' configure.ac || die
+
+	# Only build the Displaylink test program, if a user want's it.
```

Comment:
no apostrophe in `wants`, same on next line ;)

Reply:
Whoops ;-) Fixed.